cordova/cordova.go

```diff
@@ -67,6 +67,7 @@ func (builder *Model) commandSlice(cmd ...string) []string {
 		if builder.buildConfig != "" {
 			cmdSlice = append(cmdSlice, "--buildConfig", builder.buildConfig)
 		}
+		cmdSlice = append(cmdSlice, builder.customOptions...)
 	}
 
 	cmdSlice = append(cmdSlice, builder.customOptions...)
```

Comment:
`cmdSlice = append(cmdSlice, builder.customOptions...)` which one is required? Is it possible that the input is not processed correctly, this is why there is no `customOptions` appended?